src/ChipInput.js

```diff
@@ -158,7 +158,7 @@ class ChipInput extends React.Component {
       }
     }
 
-    this.autoComplete.handleItemTouchTap = (event, child) => {
+    this.autoComplete.handleItemClick = (event, child) => {
```

Comment:
In order to keep compatible with v0.19.x, maybe we should also assign `this.autoComplete.handleItemTouchTap = this.autoComplete.handleItemClick` after that? :thinking: